dao/searchDao.js

```diff
@@ -1,21 +1,15 @@
 const db = require('./db');
-let sqliteDatabase;
-if(process.env.NODE_ENV === 'test') {sqliteDatabase = require('better-sqlite3');}
-
+const allFilters = require('./allFilters');
+let sqliteDatabase = (process.env.NODE_ENV === 'test') ? require('better-sqlite3') : null;
 exports.getAll = function(filter, cb) {
-    let classFilter = '';
-    if(filter.class) {
-        classFilter = ' AND class = ' + db.escape(filter.class);
-    }
-    let sqlQuery = 'SELECT * FROM tbl_tickets WHERE reserver IS NULL' + classFilter + ';';
-
-    if(process.env.NODE_ENV === 'test') {
-        // Please look at the README for debugging to the console
-        const Testdb = new sqliteDatabase('airlinesTest.db', {memory: true});
-        const result = Testdb.prepare(sqlQuery).all();
+    let sqlQuery = 'SELECT flight, seat_row, seat, class, reserver, price, reservation_timeout, booking_id, departure, destination, departure_date, arrival_date, flight_number FROM tbl_tickets AS t LEFT JOIN tbl_flights AS f ON t.flight = f.id WHERE reserver IS NULL' +
```

Comment:
If you put the columns you want into an array, you could use the `.join` method to produce the comma-separated form more concisely than writing it all out yourself.

Reply:
Yes, however, it would break my code that I have.

Reply:
How?

```javascript
const sqlQueryFields = ['flight', 'seat_row', 'seat', 'class', 'reserver', 'price', 'reservation_timeout', 'booking_id', 'departure', 'destination', 'departure_date', 'arrival_date', 'flight_number'];
let sqlQuery = 'SELECT ' + sqlQueryFields.join(', ') + FROM tbl_tickets AS t ...
```

(OK, that's not actually any more concise as a whole, but it makes that *line* more concise.)